apsuite/commisslib/measure_bbb_data.py

```diff
@@ -523,3 +523,250 @@ def waterfall_plot(axis, xs, zs, data):
         axis.set_ylim3d(xs.min(), xs.max())
         axis.set_xlim3d(zs.min(), zs.max())
         axis.set_zlim3d(0, data.max())
+
+
+class TuneShiftParams(_ParamsBaseClass):
+    """."""
+
+    TIME_REV = 864 / 499666000  # s
+    WAIT_INJ = 0.2  # s
+    DEF_TOL_CURRENT = 0.01  # mA
+
+    def __init__(self):
+        """."""
+        super().__init__()
+        self.plane = 'H'  # 'H' or 'V'
+        self.kickh = -25/1000  # mrad
+        self.kickv = +20/1000  # mrad
+        self.wait_bbb = 9  # s
+        self.currents = list()
+        self.filename = ''
+
+    def __str__(self):
+        """."""
+        dtmp = '{0:10s} = {1:9d}  {2:s}\n'.format
+        ftmp = '{0:10s} = {1:9.3f}  {2:s}\n'.format
+        ltmp = '{0:6.3f},'.format
+        stg = f"{'plane':10s} = {self.plane:9s}\n"
```

Comment:
maybe an extra  string `"   ('H', 'V' or 'HV')"` explaining the options would be nice.

Reply:
Ok I agree. Done!